projects/rocprofiler-systems/source/lib/core/config.cpp

```diff
@@ -315,6 +316,10 @@ configure_settings(bool _init)
     ROCPROFSYS_CONFIG_SETTING(bool, "ROCPROFSYS_USE_ROCPD", "Enable rocpd backend", false,
                               "backend", "rocpd");
 
+    ROCPROFSYS_CONFIG_SETTING(bool, "ROCPROFSYS_TRACE",
+                              "Enable perfetto with trace cache", _default_perfetto_v,
```

Comment:
Optional: 
It think we try to keep the wording consistent with the other backends. Since this is the new default, the user doesn't need to know it's anything different. We can add that detail in the CHANGELOGS and Release Notes.

```suggestion
                              "Enable perfetto backend.", _default_perfetto_v,
```